connector_voicent/__manifest__.py

```diff
@@ -0,0 +1,36 @@
+# @TODO: To use the right License.
+# @TODO: Description, Website, Contributros..etc, of this module to be adjusted later. 
+{
+    'name': 'Voicent Connector',
+    'version': '12.0.1.0.0',
+    'category': 'Phone',
+    'license': 'AGPL-3',
+    'summary': """
+Voicent Connector
+=========
+Usage:
+    Go to Connectors > Backends > Voicent Backends
+    Create a new Voicent Backend with the host and port
+    Create Call Lines to determine when (which stage in the process) calls are added to the queue
+    Create Time Line to determine when (what time) calls are made
+""",
```

Comment:
@YounessMaafi Runbot is complaining about the summary being too long. You are limited to 256 characters.

Just put: `'summary': "Automate phone calls based on stage changes"`